.github/workflows/ci-checks.yml

```diff
@@ -78,3 +78,31 @@ jobs:
             env:
                 TEST_TOKEN: ${{ secrets.TEST_TOKEN }}
             run: "python -m pytest --verbose tests/"
+    validate-documentation-examples:
+        runs-on: [ ubuntu-latest ]
+        needs: [black, pylint, flake8]
+        name: "Validate examples in documentation"
+        steps:
+        -   uses: actions/checkout@v2
+        -   name: "Install dependencies"
+            run: |
+                pip install notebook tqdm ipywidgets
+                pip install -r requirements.txt
+        -   name: "Setup MVG"
+            run: |
+                pip install .
+        -   name: "Run all examples"
+            env:
+                TEST_TOKEN: ${{ secrets.TEST_TOKEN }}
+                EXAMPLES_DIR: 'docs/source/content/examples'
+            run: |
+                echo "Running notebooks containing examples"
+                jupyter nbconvert --execute --to notebook --inplace ${{env.EXAMPLES_DIR}}/check_version.ipynb \
+                                                                    ${{env.EXAMPLES_DIR}}/brief_overview.ipynb \
+                                                                    ${{env.EXAMPLES_DIR}}/sources_and_measurements.ipynb \
+                                                                    ${{env.EXAMPLES_DIR}}/tabular_example.ipynb \
+                                                                    ${{env.EXAMPLES_DIR}}/error_handling.ipynb \
+                                                                    ${{env.EXAMPLES_DIR}}/analysis_classes.ipynb \
+                                                                    ${{env.EXAMPLES_DIR}}/analysis_visual.ipynb \
+                                                                    ${{env.EXAMPLES_DIR}}/labeling.ipynb
+                echo "Done"
```

Comment:
Let's turn this into a bash script that finds all notebooks in the folder and executes them. This can be used by the user as well locally to update the notebooks before pushing it.